src/views/Layout.css

```diff
@@ -18,14 +18,58 @@
 }
 
 .Layout-header {
-	background-color: var(--color-bg);
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background: var(--color-white);
 	padding-bottom: 0.5rem;
 	padding-top: max(env(safe-area-inset-top), 1rem);
 	text-align: center;
+	height: 55px;
+}
+
+.Layout-header-name {
+	margin: 0;
+	color: var(--color-branding);
+	font-size: 2rem;
+	font-family: 'Righteous', sans-serif;
+	text-decoration: none;
+}
+
+.Layout-header-login {
+	display: flex;
+	background-color: var(--btn-bg-color-confirm);
+	border-radius: 2rem;
+	max-width: 145px;
+	padding: 2px;
 }
 
-.Layout-header > h1 {
+.Layout-header-login:hover {
+	background: var(--btn-bg-color-confirm-hover);
+}
+
+.Layout-header-signed-in {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.Layout-header-profile {
+	border-radius: 50%;
+	border: 1px solid transparent;
+	width: 40px;
+}
+
+.Layout-header button {
+	background: transparent;
+	color: var(--color-text);
```

Comment:
Looking at the deployed it appears that this overrides the button styles in index making the text in the signin/signout button black. Suggestion is to remove this line

Reply:
Thanks for catching that. I'll change the text color of 'button' to '--color-white' so it applies to all the buttons in both light and dark mode.